### ISSUE TITLE
gff header present, causing problems with chain_samples.py

### ISSUE THREAD

**Issue by williamrowell** [Magdoll/cDNA_Cupcake#13]
During the sanity check, `chain_samples.py` uses GFF.collapseGFFReader to process the gff.  This function fails if there is a header.
```
  File "/home/wrowell/anaconda2/lib/python2.7/site-packages/cupcake-3.1-py2.7-linux-x86_64.egg/EGG-INFO/scripts/chain_samples.py", line 19, in sample_sanity_check
    ids2 = [r.seqid for r in GFF.collapseGFFReader(gff_filename)]
  File "/home/wrowell/anaconda2/lib/python2.7/site-packages/cupcake-3.1-py2.7-linux-x86_64.egg/cupcake/io/GFF.py", line 386, in next
    return self.read()
  File "/home/wrowell/anaconda2/lib/python2.7/site-packages/cupcake-3.1-py2.7-linux-x86_64.egg/cupcake/io/GFF.py", line 543, in read
    assert raw[2] == 'transcript'
IndexError: list index out of range
```
New PB gff files have the following header:
```
##gff-version 3
```

**Comment by Magdoll** [comment]
both this issue and issue 12 are due to not reading variable header lines that start with '#'. This should be fixed in Cupcake version 3.2. I'll wait for @williamrowell to confirm.

**Comment by williamrowell** [comment]
Fixed in 224812a89a8d5892b99ed9e3ab467165191f0e3f.  Thanks @Magdoll
